src/Core/Domain/Order/QueryResult/OrderProductForViewing.php

```diff
@@ -225,7 +225,7 @@ public function __construct(
         array $packItems = [],
         ?OrderProductCustomizationsForViewing $customizations = null,
         string $mpn = '',
-        ?int $shipmentId = null
+        ?array $shipmentIds = []
```

Comment:
```suggestion
        array $shipmentIds = []
```
Every time is array no ?